### PR TITLE
plot_chains implementation

### PR DESCRIPTION
**Pull request by Vaibhavdixit02**

### CODE REVIEW

**Comment by Vaibhavdixit02** [comment]
I think Travis needs to be re-run for this?

**Comment by Vaibhavdixit02** [comment]
@ChrisRackauckas I think you missed this.

**Comment by ChrisRackauckas** [comment]
You should remake this into a Plots.jl recipe so that it matches the rest of DiffEq. See the type recipes here:

http://docs.juliaplots.org/latest/recipes/

File: src/utils.jl [hunk 1]
```diff
@@ -0,0 +1,11 @@
+using Mamba
```
Comment (ChrisRackauckas) [inline src/utils.jl]: These aren't needed since it's inherited from the module it's in.

**Pull request merged**